wesnoth-ei.po

```diff
@@ -207,9 +208,9 @@ msgid ""
 "outpost, he sent Gweddry, who would be accompanied by Dacyn the mage. Our "
 "story begins there."
 msgstr ""
-"Do severní základny poslal Owaeca ze šlechtického rodu. Do jížní pevnosti "
-"poslal Gweddraho, kterého měl doprovázet mág Dacyn. Náš příběh začíná právě "
-"zde."
+"Do severní základny poslal Owaeca ze šlechtického klanového rodu. Do jižní "
```

Comment:
Owaeca, šlechtice klanu?

Reply:
To mi tedy nepřipadá o moc lepší.

Reply:
netrvám na tom

jen se může stát, že na to zapomenu, narazím při hře, praští mne to do očí a opravím to, tak se pak ozvěte